frontend/src/tenants/FeatureToggle/FeatureToggle.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { useTenant } from 'tenants';
+
+/**
+ * Feature Toggle properties.
+ */
+interface Props {
+  tenant: string;
+  hide?: boolean;
+}
+
+/**
+ * The FeatureToggle component provides a way to hide or show UI features based on the current tenant.
+ * @param param0 FeatureToggle component properties.
+ * @returns If the current tenant matches the `tenant` property it will render the children; otherwise returns `null`.
+ */
+const FeatureToggle: React.FC<Props> = ({ tenant, hide = false, children }) => {
```

Comment:
Is this component used just to hide children components based on the tenant?  Essentially a wrapper component for other components to show/hide things?

Reply:
yes

Reply:
Then use the tenant hook to determine whether it should hide/show `useTenant()`

Reply:
yeah - that is what the component is doing (I think). Happy to jump on a quick call to discuss

Reply:
Ah I think I'm misunderstanding.  You place the tenant key on the `FeatureToggle` to identify which tenant this feature belongs to?

Reply:
correct